### PR TITLE
Added scripts for covering [Policies] SDL.ActivateApp from HMI and 'isPermissionsConsentNeeded' parameter in the response

### PR DESCRIPTION
**Pull request by aarotar**

### CODE REVIEW

**Comment by aarotar** [comment]
@vvakulenko please review

**Comment by aarotar** [comment]
@aderiabin please review

**Comment by aarotar** [comment]
@vvakulenko please review

File: test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_false.lua [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+-- Requirement summary: 
+--    [Policies] SDL.ActivateApp from HMI and 'isPermissionsConsentNeeded' parameter in the response
+--
+-- Expected result:
+--      On receiving SDL.ActivateApp PoliciesManager must respond with "isPermissionsConsentNeeded:false" to HMI, consent for custom permissions is not appeared
+---------------------------------------------------------------------------------------------
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+local mobile_session = require('mobile_session')
```
Comment (aderiabin) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_false.lua]: @aarotar please avoid unused variables (mobile_session variable isn't used in script)

File: test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_true.lua [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+commonSteps:DeactivateAppToNoneHmiLevel()
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+commonFunctions:userPrint(34, "Test is intended to check that isPermissionsConsentNeeded:true for app permissions that require consent")
+
+function Test:PTU_with_app_permissions_require_consent()
+	local RequestIdGetURLS = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+	EXPECT_HMIRESPONSE(RequestIdGetURLS,{result = {code = 0, method = "SDL.GetURLS", urls = {{url = "http://policies.telematics.ford.com/api/policies"}}}})
+	:Do(function(_,data)
```
Comment (aderiabin) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_true.lua]: @aarotar please avoid unused variables (data variable isn't used in script)

File: test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_true.lua [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+		self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
+			{
+				requestType = "PROPRIETARY",
+				fileName = "filename"
+			}
+		)
+		EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "PROPRIETARY" })
+		:Do(function(_,data)
```
Comment (aderiabin) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_true.lua]: @aarotar please avoid unused variables (data variable isn't used in script)

File: test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_true.lua [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+				local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest",
+					{
+						fileName = "PolicyTableUpdate",
+						requestType = "PROPRIETARY"
+					}, "files/PTU_NewPermissionsForUserConsent.json")
+				local systemRequestId
+				EXPECT_HMICALL("BasicCommunication.SystemRequest")
+				:Do(function(_,data)
```
Comment (aderiabin) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_true.lua]: @aarotar in this case you have shadowed value of argument "data" of parent function. Please avoid shadowing arguments, use a new name for this argument.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_true.lua [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+					systemRequestId = data.id
+					self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate",
+					{
+						policyfile = "/tmp/fs/mp/images/ivsu_cache/PolicyTableUpdate"
+				})
+				function to_run()
```
Comment (aderiabin) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_true.lua]: @aarotar please avoid global variable creation, create local variables instead global ones

File: test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_true.lua [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+function Test:Activate_app_isPermissionsConsentNeeded_true()
+	local RequestId = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.HMIAppID })
+	EXPECT_HMIRESPONSE(RequestId)
+	:Do(function(_,data)
+		if data.result.isPermissionsConsentNeeded ~= true then
+			commonFunctions:userPrint(31, "Wrong SDL behavior: isPermissionsConsentNeeded should be false for app permissions that require consent")	
+		else
+		    commonFunctions:userPrint(33, "isPermissionsConsentNeeded is true for app with permissions that require consent - expected behavior")
+		    local RequestId = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage",
```
Comment (aderiabin) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_true.lua]: @aarotar please avoid shadowing variables, use a new name for new variables.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_true.lua [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+	EXPECT_HMIRESPONSE(RequestId)
+	:Do(function(_,data)
+		if data.result.isPermissionsConsentNeeded ~= true then
+			commonFunctions:userPrint(31, "Wrong SDL behavior: isPermissionsConsentNeeded should be false for app permissions that require consent")	
+		else
+		    commonFunctions:userPrint(33, "isPermissionsConsentNeeded is true for app with permissions that require consent - expected behavior")
+		    local RequestId = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", 
+			{language = "EN-US", messageCodes = {"DataConsent"}})
+			EXPECT_HMIRESPONSE(RequestId)
+			:Do(function(_,data)
```
Comment (aderiabin) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_true.lua]: @aarotar please avoid unused variables (data variable isn't used in script)

**Comment by aarotar** [comment]
@vvakulenko please review last version of scripts

**Comment by aarotar** [comment]
@aderiabin please review last version of scripts

**Comment by aarotar** [comment]
@maria-mih, please review

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**